### DISCUSSION TITLE
PRs in Bitbucket Server: don't show dependancies updates info like age/adoption/passing/confidence in PRs like github

### DISCUSSION THREAD

**Discussion by catsonstar** [renovatebot/renovate#18256] [Request Help] [answered]

**Comment by catsonstar** [comment]
Sorry, cannot edit above content. Trying to ask question here: Is it possible to show dependancies updates info like age/adoption/passing/confidence in Bitbucket PR like in Github? I believe these info are quite helpful to make the decision to merge the PRs or not.

**Reply by rarkins** [reply]
Try adding `github>whitesource/merge-confidence:beta` to your `extends` array

**Reply by rarkins** [reply]
https://github.com/whitesource/merge-confidence

**Reply by catsonstar** [reply, marked as answer]
thanks so much @rarkins. I tried that, still cannot get merge confidence info in my BitBucket PR. then check further, found the renovate.json inside my repository overwrite the config.js. Now it works! thanks so much~
